unittest/test.js

```
@@ -0,0 +1,16 @@
+// Mock
+document = {
```

Biome (1.9.4)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)